### PR TITLE
Bug fix for building height in LCZ urban parameter table

### PR DESCRIPTION
**Pull request by cenlinhe**
Bug fix for building height in LCZ urban parameter table

TYPE: bug fix

KEYWORDS: urban, BEP, BEP_BEM, parameter table, building height

SOURCE: Cenlin He (NCAR), Alberto Martilli (CIEMAT, Spain)

DESCRIPTION OF CHANGES:
Problem:
The BEP and BEP_BEM urban schemes (urban_physics = 2,3) require the building height distribution to be a multiple of 5-m because of the hard-coded dz_u = 5 parameter in BEP and BEP_BEM source code. However, currently the urban parameter table (URBPARM_LCZ.TBL) has a building height distribution as a multiple of 3-m, which will cause model issues.

Solution:
Change the building height distribution in URBPARM_LCZ.TBL to be a multiple of 5-m. In addition to the model code requirement as mentioned above, the justifications of this change also include: (1) the NUDAPT database has building heights as a multiple of 5m. (2) the updated mean building heights in URBPARM_LCZ.TBL aer equal to the middle range values of the urban LCZs from Stewart and Oke (2012): Stewart, I. D., & Oke, T. R. (2012). Local climate zones for urban temperature studies. Bulletin of the American Meteorological Society, 93(12), 1879-1900.

LIST OF MODIFIED FILES:
run/URBPARM_LCZ.TBL

TESTS CONDUCTED: 
1. The mods fixed the problem.

RELEASE NOTE: This is a bug fix for building height in LCZ urban parameter table

### CODE REVIEW

**Comment by weiwangncar** [comment]
The regression test results:

    Test Type              | Expected  | Received |  Failed
    = = = = = = = = = = = = = = = = = = = = = = = =  = = = =
    Number of Tests        : 23           24
    Number of Builds       : 60           57
    Number of Simulations  : 158           150        0
    Number of Comparisons  : 95           86        0

    Failed Simulations are: 
    None
    Which comparisons are not bit-for-bit: 
    None

File: run/URBPARM_LCZ.TBL [hunk 1]
```diff
@@ -543,105 +543,103 @@ BUILDING HEIGHTS: 1
...
 #      (sf_urban_physics=2,3)
 
 #     height   Percentage
 #      [m]       [%]
-      24.0      100.0
+      40.0      17.
+      45.0      21.
+      50.0      24.
+      55.0      21.
+      60.0      17.
```
Comment (dudhia) [inline run/URBPARM_LCZ.TBL:580]: These don't look similar to the previous ones.

File: run/URBPARM_LCZ.TBL [hunk 2]
```diff
@@ -543,105 +543,103 @@ BUILDING HEIGHTS: 1
 
 #     height   Percentage
 #      [m]       [%]
-      21.0      100.0
+      40.0      17.
+      45.0      21.
+      50.0      24.
+      55.0      21.
+      60.0      17.
```
Comment (dudhia) [inline run/URBPARM_LCZ.TBL:550]: this is another category that changed a lot

**Comment by weiwangncar** [comment]
@cenlinhe Do you want this to be in 4.5.1?

**Comment by cenlinhe** [comment]
Yes, we want to include this bug fix as soon as possible.

**Review by dudhia — APPROVED** [review]

**Review by smileMchen — APPROVED** [review]

**Pull request merged**